sdks/core/src/main/java/org/gbif/pipelines/core/parsers/taxonomy/TaxonRecordConverter.java

```diff
@@ -6,30 +6,26 @@
 import java.util.stream.Collectors;
 import lombok.AccessLevel;
 import lombok.NoArgsConstructor;
-import org.gbif.pipelines.io.avro.Diagnostic;
-import org.gbif.pipelines.io.avro.MatchType;
-import org.gbif.pipelines.io.avro.Nomenclature;
-import org.gbif.pipelines.io.avro.Rank;
-import org.gbif.pipelines.io.avro.RankedName;
-import org.gbif.pipelines.io.avro.Status;
-import org.gbif.pipelines.io.avro.TaxonRecord;
-import org.gbif.rest.client.species.IucnRedListCategory;
-import org.gbif.rest.client.species.NameUsageMatch;
-
-/** Adapts a {@link NameUsageMatch} into a {@link TaxonRecord} */
+import org.gbif.pipelines.io.avro.*;
+import org.gbif.rest.client.species.NameUsageMatchResponse;
+
+/** Adapts a {@link NameUsageMatchResponse} into a {@link TaxonRecord} */
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class TaxonRecordConverter {
 
+  public static final String IUCN_REDLIST_GBIF_KEY = "19491596-35ae-4a91-9a98-85cf505f1bd3";
```

Comment:
will this never change?

Reply:
changed to use gbif-api constants - same as GBIF Nub UUID reference